src/main/java/org/opensearch/security/securityconf/impl/v7/TenancyConfigV7.java

```diff
@@ -0,0 +1,16 @@
+package org.opensearch.security.securityconf.impl.v7;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+public class TenancyConfigV7 {
```

Comment:
Why the name V7 here? There is only one version of this class. We can move this class out of v7 folder and rename it to `TenancyConfig`